tikv-client/src/main/java/com/pingcap/tikv/predicates/ScanAnalyzer.java

```diff
@@ -153,6 +153,7 @@ public ScanPlan buildScan(List<TiColumnInfo> columnList, List<Expression> condit
       // table name, index and handle column
       int indexSize = index.getIndexColumns().size() + 2;
       if (isDoubleRead) {
+        cost *= 4; // we assume double read's cost is around 4 times of simple table scan.
```

Comment:
Could you plz explain why the factor is multiplied over the whole cost here?

From my perspective this factor should be EITHER multiplied over tableSize (indicating IO for each table row) OR indexSize (IO for each index row).

Reply:
If a factor is applied to `indexSize`, the same factor should be applied to the `indexSize` in the else-clause